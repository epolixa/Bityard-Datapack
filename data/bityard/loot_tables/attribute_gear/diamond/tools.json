{
	"pools": [
		{
			"rolls": 1,
			"entries": [
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/tools/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/tools/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/tools/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/tools/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/tools/swift"
				}
			]
		}
	]
}