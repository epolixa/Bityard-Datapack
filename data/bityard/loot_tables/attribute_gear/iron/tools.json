{
	"pools": [
		{
			"rolls": 1,
			"entries": [
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/tools/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/tools/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/tools/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/tools/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/tools/swift"
				}
			]
		}
	]
}