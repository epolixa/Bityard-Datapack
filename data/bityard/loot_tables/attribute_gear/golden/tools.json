{
	"pools": [
		{
			"rolls": 1,
			"entries": [
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/tools/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/tools/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/tools/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/tools/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/tools/swift"
				}
			]
		}
	]
}